src/styles/stars-block/stars.scss

```diff
@@ -0,0 +1,76 @@
+.star {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 23px;
+}
+
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url("../images/star.svg");
+    background-repeat: no-repeat;
+    background-position: center;
+    width: 16px;
+    height: 16px;
+  }
+
+  &__star:not(:last-child) {
+    margin-right: 4px;
+  }
+}
+
+.stars--1 li:nth-child(-1n + 1)::after {
```

Comment:
all such styles seems redundant
just reuse code from the previous task

Reply:
![Screenshot_7](https://github.com/mate-academy/layout_catalog/assets/149514552/5c616e38-90c1-4a88-b65f-ecaffa882608)